src/amplitude/amplitude.ts

```diff
@@ -0,0 +1,62 @@
+import * as amplitude from '@amplitude/analytics-browser';
+import {track} from '@amplitude/analytics-browser';
+import {erProd} from '../utils/url-utils';
+import {AmplitudeEvent} from './taxonomy-events';
+
+export function initAmplitude() {
+    const apiKey = erProd() ? '691963e61d2b11465aa96acfcaa8959b' : 'faf28eb5445abfe75c7ac28ae7a8d050';
```

Comment:
I am only not sure if this code should be in code...everything else is good

Reply:
It is in code in veilarbpersonflatefs and in aktivitetsplanen etc.
The key here is to say which project the data is going to (meaning that we can choose PO Arbeidsoppfølging - dev or  PO Arbeidsoppfølging - prod  when we are visulizing data)

Reply:
It's no problem, it's only pointing to a specific project (should maybe be called projectID?)!
[Ref. Aksel](https://aksel.nav.no/god-praksis/artikler/mal-brukeratferd-med-amplitude): "Åpne Amplitude API nøkler til Amplitude prosjektet deres kan trygt legges til i klartekst i sporingskoden."